### PR TITLE
Stage 0.10.1.

### PR DESCRIPTION
**Pull request by anarkiwi**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ services:
             context: api
```
Comment (cglewis) [inline docker-compose.yaml]: remove build section

File: docker-compose.yaml [hunk 2]
```diff
@@ -55,7 +55,7 @@ services:
             context: .
```
Comment (cglewis) [inline docker-compose.yaml]: remove build section

**Comment by cglewis** [comment]
workers.json and changelog need to be updated

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+# v0.11.2 (2020-02-13)
```
Comment (cglewis) [inline CHANGELOG.md]: wrong version

**Comment by anarkiwi** [comment]
> workers.json and changelog need to be updated

updated.

File: CHANGELOG.md (hunk 3 above)
Comment (cglewis) [reply CHANGELOG.md]: can you also add docker support for armv7 and arm64 for rabbitmq, redis, poseidon, poseidon-api, and workers images?

Comment (anarkiwi) [reply CHANGELOG.md]: Fixed.

**Pull request merged**